bag/objectstore/objectstore.py

```diff
@@ -132,6 +132,10 @@ def download_file(connect, container_name, file_path, target_path=None, target_r
         os.utime(newfilename, (epoch_modified, epoch_modified))
 
 
+def download_file_data(connect, container_name, file_path):
```

Comment:
Als deze alleen wordt gebruikt voor de combi `container = 'productie' path = f'wkpb/brondocumenten/{brondocument.documentnaam}' connection = 'GOB_user'`, is het dan niet handiger om dat hier neer te zetten en de methode download_wkpb_file te noemen?

Reply:
Bij testen en ontwikkeling heb ik wel acceptatie ipv productie gebruikt. Het pad wordt  ook gebruikt voor andere files en GOB_user kan Bag_user zijn voor niet GOB_import

Reply:
De methode wordt maar op precies 1 plek aangeroepen, dus hoe generiek ook, hergebruikt is hij niet (en er zijn andere methodes voor andere configuraties, zie `def download_diva_file` er direct onder).

Los daarvan vind ik welke user je nodig hebt, wat het container en pad is, niet configuratie van de API-viewset `BroncodeView`, maar van de `objectstore` functionaliteit. Stel, de methode wordt wel hergebruikt, en de inrichting van de objectstore wijzigt, dan zou je de op allerlei plaatsen snippets moeten wijzigen ipv alleen in bestand `objectstore.py`.